TLM/TLM/UI/SubTools/TTL/TimedTrafficLightsTool.cs

```diff
@@ -1699,9 +1690,9 @@ public TimedTrafficLightsTool(TrafficManagerTool mainTool)
                                 }
 
                                 var infoRect = new Rect(
-                                    offsetScreenPos.x + (modeWidth / 2f) +
-                                    (7f * zoom * (numInfos + 1)) + (infoWidth * numInfos),
-                                    offsetScreenPos.y - (infoHeight / 2f),
+                                    offsetScreenPos.x + modeWidth / 2f +
```

Comment:
[Note]
I think your auto code cleanup has been setup to do some extra stuff. 
I also think its `.editorConfig` fault for not explicitly handling this situations and leaving it to the person's IDE settings.

Reply:
it just means there is more code to review.

Reply:
and increases chance of conflict.

Reply:
Yes, I did not tell Visual Studio to do any of that, nor did I explicitly configure it to happen automatically. It's extremely frustrating. But I can go back and revert these if y'all believe it's worthwhile.

Reply:
No I don't think its worth while.
But maybe you can play with .editorconfig not to do this. But I think that's outside of scope of this PR

Reply:
![image](https://user-images.githubusercontent.com/26344691/160305736-72d540b5-3491-44db-8a7b-aced4d358a51.png)
for some reason you have to use text editor to modify values.